### PR TITLE
Ossm 1912

### PR DESCRIPTION
**Pull request by fjglira**
Adding again the test case about the must gather tool, adding new env variable to set a new image tag when is needed. The test will ve active again

### CODE REVIEW

**Review by mattmahoneyrh — COMMENTED** [review]
Consider parameterizing "2.3", and in stead use MUSTGATHERTAG

File: tests/test.env [hunk 1]
```diff
@@ -4,3 +4,4 @@ export GODEBUG="x509ignoreCN=0"
 export SAMPLEARCH=x86
 export ROSA=false
 export TEST_GROUP=full
+export MUSTGATHERTAG=2.3
```
Comment (fjglira) [inline tests/test.env:7]: @mattmahoneyrh

**Review by fjglira — COMMENTED** [review]
@mattmahoneyrh

File: pkg/ossm/smcp_must_gather.go [hunk 2]
```diff
@@ -24,6 +24,7 @@ import (
 )
 
 var mustGatherImage = "registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8"
+var mustGatherTag string = util.Getenv("MUSTGATHERTAG", "2.3")
```
[the remark this answers is not in the document]
Comment (fjglira) [reply pkg/ossm/smcp_must_gather.go:28]: Agree that 2.3 will be the default version from now

**Review by mattmahoneyrh — APPROVED** [review]

**Pull request merged**